build/push-aws.sh

```diff
@@ -38,6 +38,6 @@ function push_one_binary {
   rm -f ${tmpfile}
 }
 
-for proj in blocks; do
-  push_one_binary ${proj}/${proj}
+for i in "${PROJECTS[@]}"; do
```

Comment:
bah, no need for an array, I'd just list them out in the for loop directly.
see: https://github.com/cockroachdb/examples-go/blob/master/build/push-aws.sh